app/views/components/pageRegistration.html

```diff
@@ -15,12 +15,14 @@ <h2 class="page-title border" ng-bind="page.title"></h2>
       <div class="col-sm-9 col-xs-7">
         <button class="btn btn-success btn-important btn-lg btn-block"
                 type="submit"
-                ng-disabled="savingAnswers"
+                ng-disabled="savingAnswers || !pageIsValid(page.id)"
                 ng-click="goToNext()">
           <span ng-show="!savingAnswers && !nextPage()" translate>Continue</span>
           <span ng-show="!savingAnswers && nextPage()" translate>Next Page</span>
           <span ng-show="savingAnswers" translate>Saving...</span>
         </button>
+
+        <p ng-if="!pageIsValid(page.id)" style="color:red;" class="text-center" translate>Please fill in all required fields.</p>
```

Comment:
It may be a better idea to move style to a css file.

Reply:
Looks like there's already a class for it. https://github.com/CruGlobal/conf-registration-web/blob/master/app/styles/variables.scss#L21
Hopefully this works:
```suggestion
        <p ng-if="!pageIsValid(page.id)" class="red text-center" translate>Please fill in all required fields.</p>
```